src/shared/icons/times-white.svg

```diff
@@ -0,0 +1,10 @@
+<svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Maybe let's just call it for its use, so `close`? Also, what's the difference between this and regular `times`? Do we need both?